models/src_shopify.yml

```
@@ -1340,7 +1340,7 @@ sources:
           - name: created_at
             description: When the checkout discount application was created.
           - name: discount_id
-            description: ID of the discount.
+            description: ID of the discount. Use `code` instead.
```

```suggestion
            description: ID of the discount. Deprecated, use `code` instead.
```
Might be helpful to explain why we need to use code instead. You can rephrase of course!